<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Агенство недвижимости</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>651</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="flat">
     <attribute name="title">
      <string>квартиры</string>
     </attribute>
     <widget class="QTableView" name="tableView_flat">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>481</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="buyer">
     <attribute name="title">
      <string>покупатели</string>
     </attribute>
     <widget class="QTableView" name="tableView_buyer">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>481</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="sale">
     <attribute name="title">
      <string>договоры</string>
     </attribute>
     <widget class="QTableView" name="tableView_sale">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>481</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="salesman">
     <attribute name="title">
      <string>продавцы</string>
     </attribute>
     <widget class="QTableView" name="tableView_salesman">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>481</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="broker">
     <attribute name="title">
      <string>маклеры</string>
     </attribute>
     <widget class="QTableView" name="tableView_broker">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>481</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_del_sale">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>510</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_del_buyer">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>440</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_del_flat">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>370</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_del_flat">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>340</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>id квартиры</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_del_buyer">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>410</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>id  покупателя</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_del_sale">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>480</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>id договора</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_buyer">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>120</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить покупателя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_sale">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>170</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить договор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_flat">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>70</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить квартиру</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_del_flat">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>370</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>удалить квартиру</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_del_buyer">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>440</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>удалить покупателя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_del_sale">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>510</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>удалить договор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_salesman">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>220</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить продовца</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_broker">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>270</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить маклера</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
